[DOCS] Changes to the Threat Intel tab on the Alert Details pane (#1015)
elastic/security-docs: docs/detections/alerts-ui-manage.asciidoc

[float]
[[view-alert-details]]
=== View alert details

> To further inspect an alert, click the *View details* button from the Alerts table. The alert details flyout appears with several options to view alert data.

Reviewer: Tweak to help re-establish the type of alert we're talking about, since this is a long page:
```suggestion
To further inspect a detection alert, click the *View details* button from the Alerts table. The alert details flyout appears with several options to view alert data.
```

[role="screenshot"]
image::images/view-alert-details.png[Shows the Event Renderer button, 300]

The alert details flyout contains the following informational tabs:

[discrete]
*Overview*

The *Overview* tab provides an overview of the alert and shows relevant threat intelligence details. Use this information to analyze what generated the alert, so you can triage its relevance and resolve it as appropriate.

> The *Overview* tab has two main sections: *Reason* and *Document Summary*. In the *Reason* section, you'll find a custom statement describing what caused the alert and available alert details. You can use this information to quickly understand the alert's origin and to determine whether the alert is relevant to your investigation.

Reviewer 2: ```suggestion
The Overview tab contains two main sections: *Reason* and *Document Summary*. In the *Reason* section, a custom statement describes what triggered the alert and provides alert details. Use this information to understand the alert's origin and determine if the alert is relevant to your investigation.
```

> The *Overview* tab has two main parts: the *Reason* section and *Document Summary* section. In the *Reason* section, you'll find a custom statement describing what caused the alert and available alert details. You can use this information to quickly understand the alert's origin and to determine whether the alert is relevant to your investigation.

Reviewer 3: nit: 
```suggestion
The *Overview* tab has two main sections: *Reason* and *Document Summary*. In the *Reason* section, you'll find a custom statement describing what caused the alert and available alert details. You can use this information to quickly understand the alert's origin and to determine whether the alert is relevant to your investigation.
```

> TIP: You can view more details about the rule that produced the alert by clicking *View Rule detail page*.

Reviewer 2: ```suggestion
TIP: To view more details about the rule that produced the alert, click *View Rule detail page*.
```

[role="screenshot"]
image::images/reason-section.png[Shows the Reason section of an alert]

> In the *Document Summary* section, you can find general alert details, including any threat intelligence that's available for the alert. General alert details are conveyed as a curated list of fields containing essential and available information about the alert. You can use these details to start exploring what caused the alert and then continue your investigation by checking out available threat intelligence.

Reviewer 2: Be careful when using common American terminology like "checking out" - we should aim to be more direct. Also, don't know if it's worth mentioning the `+` and `-` options in the doc summary to filter the Alerts table. 
 
```suggestion
The *Document Summary* section provides general alert details, including any available threat intelligence data.  Use this information to investigate what generated the alert. 
```

Author: @joepeeples have you already doc'd, or are you planning to doc, the new `+` and `-` options somewhere within this topic? If you're not/haven't, I can insert something in the Document Summary section.

[role="screenshot"]
image::images/document-summary-section.png[Shows the Document Summary section of an alert]

> {es-sec} collects threat intelligence data for all alerts. Fields that show threat intelligence data are placed after general alert details and appended with an informational icon.

Reviewer 2: ```suggestion
{elastic-sec} collects threat intelligence data for all alerts. Fields that show threat intelligence data display an informational icon.
```

[role="screenshot"]
image::images/threat-intel-fields.png[Shows a field with threat intel]

> When gathering threat intelligence on an alert, {es-sec} queries the alert for indicator matches from the past 30 days. The query looks for the following fields:

Reviewer 2: ```suggestion
When gathering threat intelligence data for an alert, {elastic-sec} queries the alert for indicator matches from the past 30 days. The query looks for the following fields:
```

- `url.full`: The full URL of the event source
- `registry.path`: The full registry path, including the hive, key, and value

> If these fields aren't present, {es-sec} does not perform the query and threat intelligence for the alert is not displayed.

Reviewer 2: ```suggestion
If these fields aren't available, {elastic-sec} does not execute the query and threat intelligence data for the alert is not displayed.
```

Matched threats are organized into several sections, described below. Within each section, matched threats are shown in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.

* *Threat Matched Detected*: This section is solely reserved for threat indicator matches identified by an indicator match rule. Threat indicator matches are produced whenever event data matches a threat indicator field value in your indicator index. If indicator threat matches are not discovered, the section displays a message that none are available.
* *Enriched with Threat Intelligence*: This section shows indicator matches that {es-sec} found when querying the alert for fields with threat intelligence. You can use the date time picker to modify the query timeframe, which looks at the past 30 days by default. Click the **Inspect** button, located on the far right of the threat label, to view more information on the query. If threat matches are not discovered within the selected timeframe, the section displays a message that none are available.

> * *Other*: This section shows threats that do not match any threat indicators or threat intelligence fields that {es-sec} queries the alert for. If these types of threat matches are not discovered on the alert, the section does not display.

Author: Good reminder about needing a direct object after the verb! :)

- `url.full`: The full URL of the event source
- `registry.path`: The full registry path, including the hive, key, and value

If these fields aren't present, {es-sec} does not perform the query and threat intelligence for the alert is not displayed.

TIP: To learn more about the query, select the **Threat Intel** tab, then click the **Inspect** button next to the threat you want to inspect.

> Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Indicator match alerts always populate the Threat Matched Detected section under the Threat Threat Intel tab with additional threat match data.

Reviewer 4: `the Threat Threat Intel tab`

just one `Threat` should be enough 😁

Matched threats are organized into several sections, described below. Within each section, matched threats are shown in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.

* *Threat Matched Detected*: This section is solely reserved for threat indicator matches identified by an indicator match rule. Threat indicator matches are produced whenever event data matches a threat indicator field value in your indicator index. If indicator threat matches are not discovered, the section displays a message that none are available.
* *Enriched with Threat Intelligence*: This section shows indicator matches that {es-sec} found when querying the alert for fields with threat intelligence. You can use the date time picker to modify the query timeframe, which looks at the past 30 days by default. Click the **Inspect** button, located on the far right of the threat label, to view more information on the query. If threat matches are not discovered within the selected timeframe, the section displays a message that none are available.

> * *Other*: This section shows threats that do not match any threat indicators or threat intelligence fields that {es-sec} queries the alert for. If these types of threat matches are not discovered on the alert, the section does not display.

Reviewer 4: `This section shows threats that do not match any threat indicators` 
hmm I am not sure how accurate that statement is. Frankly I think it's OK to leave out the "other" section as it is primarily placed in the code base not as a product requirement but more of handling an edge case of finding threats on an alert that we as Elastic haven't enriched. So if an alert just randomly happens to have some threat enrichments on it, we don't discard them, but it is quite unlikely for enrichments to be present when we aren't adding them, if that makes sense.

Author: Makes sense, and I can remove this area entirely if we don't want to publicize it. If it's important to keep, I can reduce the text and insert it as a note after the list.

Author: You're correct! Removing it now, thanks. :)

Reviewer 3: 👍 to deleting it

> Within each section, matched threats are shown in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.

Reviewer 4: I think this information was present on line 107, so maybe it's not necessary to repeat it here.

Author: You're correct! Removing it now, thanks. :)

- `url.full`: The full URL of the event source
- `registry.path`: The full registry path, including the hive, key, and value

> If these fields aren't available, {elastic-sec} does not perform the query and does not display threat intelligence data for the alert.

Reviewer 2: Not mandatory -- but just a suggestion. 

```suggestion
If these fields aren't available, {elastic-sec} does not perform the query; therefore, does not display threat intelligence data for the alert.
```

Author: Kept as is for now.

TIP: To learn more about the query, select the **Threat Intel** tab, then click the **Inspect** button next to the threat you want to inspect.

Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Indicator match alerts always populate the *Threat Matched Detected* section under the *Threat Intel* tab with additional threat match data.

*Threat Intel*

> The *Threat Intel* tab shows the number of individual threats matching the alert. You can expand and collapse threat details by clicking the arrow icon at the end of the threat label. Each threat is labelled with values from the `matched.field` and `matched.atomic` fields and displays the threat intelligence provider. If the alert does not contain threat intelligence data, the *Threat Intel* tab displays a message that none is available.

Reviewer 2: ```suggestion
The *Threat Intel* tab shows the number of individual threats matching the alert. You can expand and collapse threat details by clicking the arrow button at the end of the threat label. Each threat is labelled with values from the `matched.field` and `matched.atomic` fields and displays the threat intelligence provider. If the alert does not contain threat intelligence data, the *Threat Intel* tab displays a message that none is available.
```

Matched threats are organized into several sections, described below. Within each section, matched threats are shown in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.

* *Threat Matched Detected*: This section is solely reserved for threat indicator matches identified by an indicator match rule. Threat indicator matches are produced whenever event data matches a threat indicator field value in your indicator index. If indicator threat matches are not discovered, the section displays a message that none are available.

> * *Enriched with Threat Intelligence*: This section shows indicator matches that {elastic-sec} found when querying the alert for fields with threat intelligence. You can use the date time picker to modify the query timeframe, which looks at the past 30 days by default. Click the **Inspect** button, located on the far right of the threat label, to view more information on the query. If threat matches are not discovered within the selected timeframe, the section displays a message that none are available.

Reviewer 2: ```suggestion
* *Enriched with Threat Intelligence*: This section shows indicator matches that {elastic-sec} found when querying the alert for fields with threat intelligence. You can use the date time picker to modify the query time frame, which looks at the past 30 days by default. Click the **Inspect** button, located on the far right of the threat label, to view more information on the query. If threat matches are not discovered within the selected time frame, the section displays a message that none are available.
```
